AssessmentService/target/classes/META-INF/maven/com.revature.caliber/AssessmentService/pom.properties

```diff
@@ -1,7 +1,7 @@
 #Generated by Maven Integration for Eclipse
-#Mon Apr 15 15:09:31 EDT 2019
+#Tue Apr 16 09:16:28 EDT 2019
 version=0.0.1-SNAPSHOT
 groupId=com.revature.caliber
 m2e.projectName=AssessmentService
-m2e.projectLocation=C\:\\Users\\Kevin\\Documents\\GithubRepository\\Project-3_Caliber-V2\\caliber-2-meta\\assessment-service\\AssessmentService
+m2e.projectLocation=D\:\\P3T3Assessment\\caliber-2-assessment-service\\AssessmentService
```

Comment:
We should make this an env variable since each person that works on it keeps changing it.